lib/plugins/slash/poker.js

```diff
@@ -0,0 +1,187 @@
+/********************************************
+ *
+ * Play a game of 5 card draw
+ *
+ * Author : Dan Schumann @dasch
+ *
+ ********************************************/
+
+var _ = require("underscore");
+
+var SECOND = 1000;
+var MINUTE = SECOND * 60;
+
+var DISCARD_HELP = 
+  '`/stuart poker discard 1 3 5` to discard your 1st, 3rd, and 5th card' + 
+  ' or `/stuart poker discard` to discard no cards. Game ends after everyone does this';
+
+// single game at a time
+var game;
+
+var Game = function(stuart, user_names, channel_name){
+  this.id = 'Game #' + (''+Math.random()).substring(10);
+  var _game = this;
+
+  // stuart is not saved to instance for garbage collection purposes
```

Comment:
@gtracy it's perhaps over-zealous to prevent memory leaks.  Sometimes saving too many references to an object caused it not to be garbage collected (`game` in this case), but I think it's no longer an issue, especially since this repo doesn't run on node 0.10 and below (someone used `Map` in their plugin)

I guess a good reason to not save a `_game.stuart = stuart` reference would be to keep the same paradigm, since `stuart` is passed every time to the `run` method, it's passed every time to the game method.  In the case where stuart perhaps disconnects and reconnects, maybe the `stuart` instance passed to `run` could be different.